hypertrace-metrics-generator/hypertrace-metrics-generator-api/src/main/java/org/hypertrace/metrics/generator/api/v1/serde/MetricSerde.java

```diff
@@ -0,0 +1,49 @@
+package org.hypertrace.metrics.generator.api.v1.serde;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.util.Map;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serializer;
+import org.hypertrace.metrics.generator.api.v1.Metric;
+
+public class MetricSerde implements Serde<Metric> {
```

Comment:
For this PR, I have added custom protobuf serde for `Metric`, `MetricIdentity`, and `OtlpMetrics`. I am facing some issues while using kafka stream protobuf serde - had logged the ticket - https://github.com/hypertrace/hypertrace/issues/312

Working on this as a follow-up. This will remove these custom objects